<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectTypesEditor</class>
 <widget class="QMainWindow" name="ObjectTypesEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Object Types Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="objectTypesLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableView" name="objectTypesTable">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget1">
       <layout class="QVBoxLayout" name="propertiesLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QtTreePropertyBrowser" name="propertiesView" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionChooseFile"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport">
   <property name="text">
    <string>Export Object Types...</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import Object Types...</string>
   </property>
  </action>
  <action name="actionChooseFile">
   <property name="text">
    <string>Choose Object Types File...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtTreePropertyBrowser</class>
   <extends>QWidget</extends>
   <header location="global">QtTreePropertyBrowser</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
